CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ These changes are available in the [master branch](https://github.com/PrefectHQ/
 - Reduces the size of the `prefecthq/prefect` Docker image by ~400MB, which is now the base Docker image used in Flows - [#1648](https://github.com/PrefectHQ/prefect/pull/1648)
 - Add a new healthcheck for environment dependencies - [#1653](https://github.com/PrefectHQ/prefect/pull/1653)
 - Add default 30 second timeout to Client requests - [#1672](https://github.com/PrefectHQ/prefect/pull/1672)
+- Add type-casting for `Parameters` - [#1682](https://github.com/PrefectHQ/prefect/pull/1682)
```

Comment:
Would you mind cutting the 0.7.0 changelog and placing this in the "unreleased" section?

Reply:
Done in #1684